src/react-apps/applications/runtime/src/components/advanced/AddressComponent.tsx

```diff
@@ -268,16 +262,7 @@ export class AddressComponent extends React.Component<IAddressComponentProps, IA
 
           <div className={'address-component-postplace-zipCode'}>
             <div className={'address-component-zipCode'}>
-              <label className={'address-component-label'}>
-                {
-                  getLanguageFromKey('ux_editor.modal_configure_address_component_zip_code', this.props.language)
-                }
-                {this.props.required ? null :
-                  <span className='address-component-label-smaller'>
-                    &nbsp;({getLanguageFromKey('general.optional', this.props.language)})
-                  </span>
-                }
-              </label>
+              {this.renderLabel('ux_editor.modal_configure_address_component_zip_code')}
```

Comment:
See that we rely on a ux_editor section in our language files, should we maybe create a section for runtime? 
Since we are going to split the dependencies runtime have our of the ini files in the future :)

Reply:
Agreed, but I think that should be done together with actually splitting the texts.